test/RelayPricer.test.ts

```diff
@@ -0,0 +1,86 @@
+import chai, { assert, expect } from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import { StubbedInstance, stubInterface } from 'ts-sinon';
+
+import { RelayPricer } from '../src';
+import { SourceApi } from '../src/types/RelayPricer';
+
+chai.use(chaiAsPromised);
+
+describe('RelayPricer', () => {
+    let fakeSourceApi: StubbedInstance<SourceApi>;
+    let pricer: RelayPricer;
+    const fakePrice1 = 23345.834630269488;
+    const fakePrice2 = 0.0775886924643371;
+    const fakePriceConversion1 = 300892.2291221775;
+    const fakePriceConversion2 = 0.0000033234490731694805;
+    const sourceCurrency = 'RBTC';
+    const targetCurrency = 'RIF';
+    const intermediateCurrency = 'USD';
+
+    describe('getPrice', async () => {
+        beforeEach(() => {
+            fakeSourceApi = stubInterface<SourceApi>();
+            pricer = new RelayPricer(fakeSourceApi);
```

Comment:
I think that we are not testing for the constructor code.

Reply:
I added an extra comment where it was tested

Reply:
updated